packages/charts-react-line/src/Line.js

```diff
@@ -11,13 +11,21 @@ export default class Line extends React.PureComponent {
     // eslint-disable-next-line react/forbid-prop-types
     data: PropTypes.array.isRequired,
     line: PropTypes.func.isRequired,
+    stroke: PropTypes.string,
   }
 
   render() {
-    const { data, line } = this.props;
+    const { data, line } = this.props,
+      stroke = this.props.stroke ? this.props.stroke : path.stroke;
```

Comment:
In this case, since we've added `stroke` to `propTypes`, should we just add the default value to `defaultProps` and then just destructure like `const { data, line, stroke }` ?